android/src/main/java/com/reactnative/googlefit/ActivityHistory.java

```diff
@@ -170,4 +175,53 @@ public ReadableArray getMoveMinutes(long startTime, long endTime, int bucketInte
         }
         return moveMinutes;
     }
+
+    public void saveWorkout(String name, String id, String description, long startTime, long endTime, float calories, Promise promise) {
+        Session session = new Session.Builder()
+                .setName(name)
+                .setIdentifier(id)
+                .setDescription(description)
+
+                .setActivity(FitnessActivities.STRENGTH_TRAINING)
```

Comment:
https://developers.google.com/android/reference/com/google/android/gms/fitness/FitnessActivities
`FitnessActivities.STRENGTH_TRAINING`; we may need a `switch case` function to get all activities depends on the user input.